core/src/test/scala/tech/beshu/ror/unit/acl/factory/decoders/definitions/LdapServicesSettingsTests.scala

```diff
@@ -199,6 +212,64 @@ class LdapServicesSettingsTests(ldapConnectionPoolProvider: UnboundidLdapConnect
           }
         )
       }
+//  Commented out until feature allowing starting up ROR without LDAP connection will be developed
```

Comment:
use `ignore` instead. It's easier to not forget about it because it's shown in test reports.